weblate/trans/forms.py

```diff
@@ -2625,6 +2626,7 @@ def __init__(
     ) -> None:
         project = kwargs.pop("project", None)
         kwargs["auto_id"] = "id_bulk_%s"
+        kwargs["initial"] = {"path": getattr(obj, "full_slug", None)}
```

Comment:
I'd rather do the condition this way:

```suggestion
        if obj is not None:
            kwargs["initial"] = {"path": obj.full_slug}
```